ayon_bin_distro/lakectlpy/wrapper.py

```diff
@@ -11,6 +12,11 @@
     import fcntl
 
 
+class DialTcpError(Exception):
+    def __init__(self, message):
+        super().__init__(message)
```

Comment:
```suggestion
class DialTcpError(Exception):
    pass
```
You're not overriding any behavior so you don't need the init implementation.